README.md

```diff
@@ -199,7 +199,8 @@ Default: `true` (always overwrite existing files)
 
 ##### `options.sourcemaps`
 
-Enables sourcemap support on files passed through the stream.  Will write inline soucemaps if specified as `true`. Uses [gulp-sourcemaps] under the hood.
+Enables sourcemap support on files passed through the stream.  Will write inline soucemaps if specified as `true`.
+Will write sourcemaps file in relative to destination folder if specified as `string`. Uses [gulp-sourcemaps] under the hood.
```

Comment:
I think it would be clearer to say "Specifying a string is shorthand for the path option"